### PR TITLE
Removed `logError` from `CBCore` and move it to `WalletLink`

### PR DESCRIPTION
**Pull request by hishbase**
Moved `logError` from `CBCore` to `WalletLink` repo

### CODE REVIEW

File: android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Observable+WalletLink.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import io.reactivex.Observable
+import timber.log.Timber
+
+
+/**
+ * Log any caught exception triggered inside an [Observable]
+ *
+ * @return The original observable
+ */
+fun <T> Observable<T>.logError(msg: String? = null): Observable<T> = doOnError {
```
Comment (johnnycoinbase) [inline android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Observable+WalletLink.kt]: add `internal`

File: android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Single+WalletLink.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import io.reactivex.Single
+import timber.log.Timber
+
+/**
+ * Log any caught exception triggered inside an [Single]
+ *
+ * @return The original single
+ */
+fun <T> Single<T>.logError(msg: String? = null): Single<T> = doOnError {
```
Comment (johnnycoinbase) [inline android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Single+WalletLink.kt]: add `internal`

File: android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Observable+WalletLink.kt (hunk 1 above)
Comment (hishbase) [reply android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Observable+WalletLink.kt]: done

File: android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Single+WalletLink.kt (hunk 2 above)
Comment (hishbase) [reply android/walletlink/src/main/java/com/coinbase/walletlink/extensions/Single+WalletLink.kt]: done

**Pull request merged**